### PR TITLE
Add authors to each file

### PR DESCRIPTION
**Pull request by aakahlow**
This PR adds the author names to each markdown file (based on the best of my knowledge).

### CODE REVIEW

File: docs/artifacts.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Authors: Ayaz Akram
```
Comment (powerjg) [inline docs/artifacts.md]: Does this syntax render correctly?

I would have expected that you had to use the yaml front matter syntax:

```
---
Authors:
  - Ayaz Akram
  - Jason Lowe-Power
---
```

I could be wrong, though :)

**Review by powerjg — COMMENTED** [review]
Other than the syntax thing, this looks fine to me!

File: docs/artifacts.md (hunk 1 above)
Comment (aakahlow) [reply docs/artifacts.md]: The markdown syntax seemed to render correctly to me. But, I liked the yaml syntax more. So, I have pushed a new commit with updated syntax.

**Comment by powerjg** [comment]
Push it!

**Pull request merged**